finetuner/experiment.py

```diff
@@ -152,10 +155,17 @@ def create_run(
             run_name=run_name,
             **kwargs,
         )
+
+        cpu = kwargs.get(CPU, True)
+        num_workers = kwargs.get(NUM_WORKERS, 4)
+
         run_info = self._client.create_run(
             run_name=run_name,
             experiment_name=self._name,
             run_config=config,
+            device='cpu' if cpu else 'gpu',
+            cpus=num_workers,
+            gpus=1,
```

Comment:
Why should we bother having `gpus` here if it's always 1?

Reply:
At some point it will be used